pages/profile.tsx

```diff
@@ -210,6 +218,49 @@ function Profile () {
         // setProfilePicture(user?.attributes.profilePicture._url);
       };
 
+    const onSubmitProfile = async (e) => {
+    
+    /*
+    ** Mapping to Moralis server
+    * varaible here -> variable in moralis _User
+    * profileName -> name
+    * profileDo -> profession
+    * profileLocation -> location
+    * profileEmail -> email
+    */  
+        e.preventDefault();
+        const name: string = profileName;
+        const profession: string = profileDo;
+        const location: string = profileLocation;
+        const email: string = profileEmail; 
+        // @ts-ignore
+        user?.set("name", name);
+        user?.set("profession", profession);
+        user?.set("location", location);
+        user?.set("email", email);
+
+        await user?.save();
+        
+        //Change text on submit button after user submits their info
+        document.getElementById("submitProfileButton").childNodes[0].nodeValue="Submitted... Go to next step!"
```

Comment:
Document query selector is fine. Might want to use https://www.tutorialspoint.com/reactjs-useref-hook